package.json

```diff
@@ -4,7 +4,8 @@
     "name": "coveo-search-ui-extensions",
     "version": "0.0.5",
     "description": "Small generic components to extend the functionality of Coveo's Search UI framework.",
-    "main": "./bin/js/commonjs/CoveoJsSearchExtensions.js",
+    "main": "./bin/commonjs/CoveoJsSearchExtensions.js",
+    "module": "./bin/es6/index.js",
```

Comment:
That is the important change.

Now it will auto switch to the es6 version when using es6 and the commonjs otherwise.

Reply:
Nice! 👍

Reply:
Link to doc for ref sake ?

Reply:
https://webpack.js.org/configuration/resolve/#resolvemainfields

C'est aussi support par Rollup.js

Reply:
Pas fan du manque d'official npm support but I can live that with that tho